bears/general/DuplicateFileBear.py

```diff
@@ -0,0 +1,32 @@
+import filecmp
+import itertools
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class DuplicateFileBear(GlobalBear):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Duplication'}
+
+    def run(self):
+        """
+        Checks for Duplicate Files using filecmp module
+        of Standard Library
+        """
+        if not self.file_dict:
+            yield Result(self, 'You did not add any file to compare')
+        else:
+            unique_tuples = [fileTuple for fileTuple in
+                             itertools.combinations(self.file_dict, 2)]
+
+            for file_pair in unique_tuples:
+                if filecmp.cmp(file_pair[0], file_pair[1]):
```

Comment:
I think we can improve in performance here. The files are already loaded by `coala`. use something like `file_dict[file_pair[0]] == file_dict[file_pair[1]]`. ? We are providing file names here, I guess python loads the files again and then compares(not sure though).